lib/vfscore/automount.c

```diff
@@ -151,51 +152,72 @@ static void vfscore_fstab_fetch_volume_args(char *v, struct vfscore_volume *vv)
 }
 #endif /* CONFIG_LIBVFSCORE_FSTAB */
 
-#ifdef CONFIG_LIBUKCPIO
-static int vfscore_mount_initrd_volume(struct vfscore_volume *vv)
+#if CONFIG_LIBUKCPIO && CONFIG_LIBRAMFS
```

Comment:
I just noticed: shouldn't that depend on `LIBVFSCORE_ROOTFS_INITRD` or `LIBVFSCORE_ROOTFS_EINITRD`?

Reply:
You want to be able to mount an initrd also through the `fstab` feature. So, as soon as RAMFS and CPIO is there, `initrd` should be supported and we need `vfscore_mount_initrd_volume()`.